src/hevm/src/EVM/Expr.hs

```diff
@@ -480,7 +491,9 @@ isLitByte _ = False
 
 -- | Returns the byte at idx from the given word.
 indexWord :: Expr EWord -> Expr EWord -> Expr Byte
-indexWord (Lit idx) (Lit w) = LitByte . fromIntegral $ shiftR w (248 - num idx * 8)
+indexWord (Lit idx) (Lit w)
+  | idx <= 31 = LitByte . fromIntegral $ shiftR w (248 - num idx * 8)
+  | otherwise = LitByte 0 -- TODO: is this correct?
```

Comment:
I've double checked and this is something that can happen in the execution of tests. Returning 0 turned out to be correct for all those tests, I left this comment just in case. We can't error here as tests will crash.